### PR TITLE
[fix] Fixing the build for OVS build for multi architecture support

### PR DESCRIPTION
**Pull request by jordanvrtanoski**
[lte][ovs] Introducing configuration to support build for ARM64 and docker

Signed-off-by: Jordan Vrtanoski <jordan.vrtanoski@gmail.com>

## Summary

Variables introduced for the build for the multi architecture support 

* `ARCH` - Architecture of the build `amd64` or `arm64`. If ommited
`arm64` is the default
* `KVERSION` - Kernel version for the `gtp` module. The default is
`4.9.0-9`

## Test Plan

Building OVS with no change in the setup of the parameters should pass.
Setting `ARCH` to `arm64` should initiate `arm64` build

## Additional Information

- [ ] This change is backwards-breaking
- [X] Mandatory for multi-platform build

### CODE REVIEW

**Review by tmdzk — APPROVED** [review]

**Comment by ekowtaylor** [comment]
@jordanvrtanoski , @tmdzk this PR has been approved, is it ready to be merged?

**Comment by ekowtaylor** [comment]
Once this PR has been approved and passing all CI checks, use the ready2merge label to indicate the maintainers can merge your PR.

**Comment by ekowtaylor** [comment]
Will need to rebase to move this forward

**Pull request closed without merging**
